manager/src/main/resources/application.properties

```diff
@@ -162,3 +162,11 @@ quarkus.log.console.json.fields.thread-id.enabled=false
 quarkus.log.console.json.fields.sequence.enabled=false
 # MGDOBR-453: Enabled logging for Workers
 quarkus.log.category."com.redhat.service.smartevents.manager.workers".level=DEBUG
+
+# MGDOBR-1009: kafka configuration for poll based error handling (errors are read from global kafka topic)
+kafka.bootstrap.servers=${event-bridge.kafka.bootstrap.servers}
+kafka.sasl.mechanism=PLAIN
```

Comment:
@kostola Can you please also set the `group.id` property to provide a consistent `consumer_group` for all instances of the Manager.

Reply:
@robpblake which `group.id` should I use? Afaik by default it uses the name of the application so it would be `manager` for every manager instance.

Reply:
@kostola `smart-events-fleet-manager` perhaps? I don't have a strong opinion, but something that makes it easy to identify the consumer group as our Fleet Manager. 

The most important thing is that the consumer group is consistent across all `Pods` of the Fleet Manager so that the topic is only processed once no matter how many pods we run for the Fleet Manager. 

cheers,

Rob

Reply:
done :+1: